### ISSUE TITLE
Assertion failed: unhandled substance: ADJUSTABLE_ATOM

### ISSUE THREAD

**Issue by phet-steele** [phetsims/states-of-matter#213]
1. Phase Changes screen
2. With something other than Adjustable Attraction selected, change the temperature
3. Select Adjustable Attraction

```
Error: Assertion failed: unhandled substance: ADJUSTABLE_ATOM
    at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/snapshot-1517131184401/assert/js/assert.js:22:13)
    at MonatomicPhaseStateChanger.setPhaseLiquid (https://bayes.colorado.edu/continuous-testing/snapshot-1517131184401/states-of-matter/js/common/model/engine/MonatomicPhaseStateChanger.js?bust=1517131895339:1431:17)
    at MonatomicPhaseStateChanger.setPhase (https://bayes.colorado.edu/continuous-testing/snapshot-1517131184401/states-of-matter/js/common/model/engine/AbstractPhaseStateChanger.js?bust=1517131895339:50:16)
    at MonatomicPhaseStateChanger.setPhase (https://bayes.colorado.edu/continuous-testing/snapshot-1517131184401/states-of-matter/js/common/model/engine/MonatomicPhaseStateChanger.js?bust=1517131895339:1376:52)
    at MultipleParticleModel.setPhase (https://bayes.colorado.edu/continuous-testing/snapshot-1517131184401/states-of-matter/js/common/model/MultipleParticleModel.js?bust=1517131895339:570:30)
    at MultipleParticleModel.initializeMonatomic (https://bayes.colorado.edu/continuous-testing/snapshot-1517131184401/states-of-matter/js/common/model/MultipleParticleModel.js?bust=1517131895339:1243:12)
    at MultipleParticleModel.initializeParticles (https://bayes.colorado.edu/continuous-testing/snapshot-1517131184401/states-of-matter/js/common/model/MultipleParticleModel.js?bust=1517131895339:730:16)
    at MultipleParticleModel.handleSubstanceChanged (https://bayes.colorado.edu/continuous-testing/snapshot-1517131184401/states-of-matter/js/common/model/MultipleParticleModel.js?bust=1517131895339:391:12)
    at Array.<anonymous> (https://bayes.colorado.edu/continuous-testing/snapshot-1517131184401/states-of-matter/js/common/model/MultipleParticleModel.js?bust=1517131895339:201:12)
    at Emitter.emit2 (https://bayes.colorado.edu/continuous-testing/snapshot-1517131184401/axon/js/Emitter.js?bust=1517131895339:177:49)
```

**Comment by jbphet** [comment]
Good catch.  This was due to changes made to #212, and I had neglected to add liquid phase initialization for the adjustable atom.  Should be fixed now.  I'll leave this open and will have it verified on upcoming RC.

**Comment by phet-steele** [comment]
Fixed in 1.1.0-rc.1.
